components/AboutPane/CommitList.tsx

```diff
@@ -15,16 +15,16 @@ export function CommitList({ className, ...props }: HTMLProps<HTMLDivElement>) {
   useEffect(() => {
     // Only reset lastVisit time if there are new commits
     if (newCount > 0) return reset;
-  }, [reset]);
+  }, [reset, newCount]);
```

Comment:
https://github.com/facebook/react/blob/main/packages/eslint-plugin-react-hooks/README.md

I know that sometimes leaving a dependency out is intentional, so please review.